man/bundle-doctor.ronn

```diff
@@ -0,0 +1,18 @@
+bundle-doctor(1) -- Checks for missing OS dependencies
+======================================================
+
+## SYNOPSIS
+
+`bundle doctor` [--quiet]
+
+## DESCRIPTION
+
+Scan the OS dependencies of each of the gems requested in your
```

Comment:
Same here as my last comment. `bundler doctor` is intended to be a general purpose "check for anything wrong about my Gemfile and gem env", not just checking OS dependencies. See #6129 as an example.

I would start off with just a general overview and then start going into the specifics about what `bundle doctor` is doing.

Reply:
Thanks again for your feedback! I explored the source code a bit and unpacked more of what `bundler doctor` checks for. Would you mind taking another look? Is there anything I'm missing, or do you think it makes sense to go further in depth on any of those items?